projects/luarocks.org/package.yml

```diff
@@ -0,0 +1,40 @@
+distributable:
+  url: https://github.com/luarocks/luarocks/archive/refs/tags/v{{version.raw}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: luarocks/luarocks/tags
+  strip: /v/
+
+dependencies:
+  lua.org: '*'
+  info-zip.org/unzip: '*'
+
+runtime:
+  env:
+    LUAROCKS_ROOT: "{{prefix}}"
+    LUA_ROOT: "{{deps.lua.org.prefix}}"
```

Comment:
this line should almost certainly be on the lua.org recipe.

Reply:
So, is this correct? (I'm new to Lua, so Idk much about it)
What change should I do?

Reply:
well, if these aren't required, then omit them. if the second one is required, I'd put it in `projects/lua.org/package.yml`. it all depends on what luarocks needs to function.

Reply:
> well, if these aren't required, then omit them. if the second one is required, I'd put it in `projects/lua.org/package.yml`. it all depends on what luarocks needs to function.

I'll omit them and see if they build.

Reply:
It works with both of them omitted.